scully/renderPlugins/puppeteerRenderPlugin.ts

```diff
@@ -63,3 +69,12 @@ export const puppeteerRender = async (route: HandledRoute): Promise<string> => {
 function waitForIt(milliSeconds) {
   return new Promise(resolve => setTimeout(() => resolve(), milliSeconds));
 }
+
+const windowSet = (page, name, value) =>
```

Comment:
This adds to the scullyVersion to the HTML? Or what element gets the attribute?

Reply:
it adds it to the body tag, as an attribute.